test/e2e/specs/specs-calypso/wp-plan-purchase__viewing-spec.js

```diff
@@ -47,20 +47,20 @@ describe( `[${ host }] Plans - Viewing: (${ screenSize }) @parallel @jetpack`, f
 		return await plansPage.waitForComparison();
 	} );
 
-	it( 'Select Business Plan', async function () {
+	it( 'Select Business plan', async function () {
 		const plansPage = await PlansPage.Expect( this.driver );
 		return await plansPage.selectPaidPlan();
 	} );
 
 	it( 'Remove any existing coupon', async function () {
-		const securePaymentComponent = await SecurePaymentComponent.Expect( this.driver );
+		const securePaymentComponent = await SecurePaymentComponent.Expect( this.this.driver );
```

Comment:
`this.this.driver` seems wrong
```suggestion
		const securePaymentComponent = await SecurePaymentComponent.Expect( this.driver );
```